pkg/domain/infra/abi/manifest.go

```diff
@@ -346,7 +346,6 @@ func (ir *ImageEngine) ManifestPush(ctx context.Context, name, destination strin
 	pushOptions.Writer = opts.Writer
 	pushOptions.CompressionLevel = opts.CompressionLevel
 	pushOptions.AddCompression = opts.AddCompression
-	pushOptions.ForceCompressionFormat = opts.ForceCompressionFormat
```

Comment:
Is this supposed to be here?

Reply:
Hmm, I think this is causing a test failure in the integration tests.

Reply:
was an accident, fixed